exercises/concept/karls-languages/.meta/src/reference/java/LanguageList.java

```diff
@@ -4,7 +4,7 @@
 public class LanguageList {
     private final List<String> languages = new ArrayList<>();
 
-    public boolean isEmpty() {
+    public boolean checkIfEmpty() {
```

Comment:
One doubt I have about this change is that it is conventional to name boolean methods `isX`. Such conventions are best taught by example, so I'm not sure I see the benefit of renaming this method.

Reply:
Ok, I will rollback that change then